README.md

```diff
@@ -25,6 +25,10 @@ After successfull installation publish the config file using one of the followin
 >
 > To migrate the image table schema into you database.
 
+Use below cammand to create storage folder 
```

Comment:
Change this line to Run below command to create symbolic link

Reply:
done